apps/cowswap-frontend/src/common/pure/CancellationModal/RequestCancellationModal.tsx

```
@@ -140,11 +140,12 @@ export function RequestCancellationModal(props: RequestCancellationModalProps):
               </p>
               <p>
                 Keep in mind a solver might already have included the order in a solution even if this cancellation is
-                successful. Read more in the{' '}
-                <HashLink target="_blank" rel="noreferrer" to={`${Routes.FAQ_TRADING}#can-i-cancel-an-order`}>
-                  FAQ
-                </HashLink>
-                .
+                successful. 
+                // Read more in the{' '}
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)